UM/Application.py

```diff
@@ -185,6 +185,9 @@ def initialize(self) -> None:
         Resources.addSecureSearchPath(os.path.join(self._app_install_dir, "Resources", self._app_name, "resources"))
 
         if not hasattr(sys, "frozen"):
+            uranium_data_root = os.environ.get('URANIUM_DATA_ROOT', None)
```

Comment:
I don't see a place where this variable is set. Is this for development environment only ? CURA_DATA_ROOT is set by conanfile.py though.

Reply:
It would allow you to use this when fooling around on a personal environment.  maybe point towards an user specific path.

The CURA_DATA_ROOT is already filled in and taken when you do a Conan install.

Reply:
Ok, just wanted to make sure it was purposely not added to conanfile.py.